app/controllers/pages_controller.rb

```diff
@@ -25,6 +25,30 @@ def bounty
     set_surrogate_key_header "bounty_page"
   end
 
+  def community_moderation
```

Comment:
Would it be helpful to create these pages in the seeds file for local environments?

Reply:
I personally think that the placeholders suffice. We don't really need these files on our local environment as its pretty much static information.